TipsForSlack.md

```diff
@@ -37,8 +37,26 @@ Please use your best judgement regarding @here and @channel. We don’t want to
+Pin an item to make a message 'stick' to a channel. Everyone can see the pinned items. They are intended for people to be able to pin messages or files that are relevant to all members of the channel. Try not to pin items that you are just trying to remember for later--if you use a pin, it can create noise for everyone else.
 
 ## Archiving & Paid Slack Plans
 
-At this point in time, we do not have the funds to pay for access to the archive of our Slack team. It costs $7/month per user. We don't qualify for the non-profit status. Any questions related to paying for a Slack plan may be directed back to this document. In order to ensure everyone has equal access, we will remain on the free plan.
+We are currently on the free Slack plan, which has a limited set of features. There have been recurring conversations over the years about whether or not we should move this workspace to a paid Slack plan. Each time these conversations have come up in the past, we have decided as a community not to move to a paid Slack plan because:
+- We hold a philosophy that no member should have to pay to be part of the Ada Network Slack group.
+- The paid plans are expensive and priced [per member](https://app.slack.com/plans/T04T5AMQ6). As Ada continues to grow, and our alum community grows, these costs would grow as well.
+- We do not qualify for the [non-profit pricing](https://slack.com/help/articles/204368833-Apply-for-the-Slack-for-Nonprofits-discount), which would allow us to have a reduced price per member, but may still be cost-prohibitive.
+- We are not willing to sacrifice the autonomy and privacy of the workspace. Specifically, we have decided against approaching Ada or another 3rd party company to sponsor the workspace for this reason.
+
+Any questions related to paying for a Slack plan may be directed back to this document.
+
+_note: Any move to a paid Slack plan will require community buy-in._
+
+### Message Retention
+
+Because we are on a free Slack plan, the admins cannot enforce a message and file [retention policy](https://slack.com/help/articles/203457187-Customize-message-and-file-retention). At the time of writing, Slack's official retention policy for free plans is:
+
+> By default, Slack will retain all messages and files (including audio and video clips) for the lifetime of your workspace.
+
+This means that even though we can only view a limited history of messages and files in our free workspace, if the workspace ever moved to a paid plan or if Slack ever increased the free plan message limit, messages that were previously inaccessible would again become accessible and searchable.
+
+If you would like to manually remove your own messages from Slack, you can do so, but you will not be able to institute a personal retention policy using Slack's retention policy settings.
```

Comment:
> If you would like to manually remove your own messages from Slack, you can do so

I had no idea someone could do this! Is there a Slack help article that explains how to do this that we could hyperlink here to explain?

Reply:
I could be wrong but I think through the UI you can only do it 1 by 1? (As long as deleting is not disabled by workspace admin, in which case you have to edit over prior messages.)

The API can be leveraged for a faster solution:
https://api.slack.com/methods/chat.delete

And I think this is the more up to date repo in python that could be used for bulk delete too:
https://github.com/sgratzl/slack_cleaner2

Reply:
Victoria had done some looking into this and found a chrome extension as well. How about this?

> If you would like to manually remove your own messages from Slack, you can do so, but you will not be able to institute a personal retention policy using Slack's retention policy settings. Some options for deleting your messages include manually clicking "Delete message" for each message in the UI, using the [Slack API](https://api.slack.com/methods/chat.delete), or leveraging a Chrome extension.